### PR TITLE
Add "Configure and link your App Clips"

### PR DESCRIPTION
**Pull request by rogerluan**
## "Configure and link your App Clips" session.

**Link:** https://developer.apple.com/videos/play/wwdc2020/10146/
**Presenters:** _Ada Chan, Luming Yin_
**Duration:** 22:56

## Two Questions:

1. Should we add the duration of every video on the top of the section? I find it useful, but it'd require everyone to do the same. Open to suggestions.
2. This is more of a hands-on video, with code samples. Should we include code samples in these summaries? For brevity reasons, I left them out. Lemme know what's the standard here (and this should probably be documented in the `CONTRIBUTING.md` file 👍 )

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -421,4 +421,52 @@ _Matt Ricketson, Taylor Kelly_
...
\ No newline at end of file
+   - Other Frameworks: **AuthenticationServices**, **AVKit**, **MapKit**, **SceneKit**, **SpriteKit**
+
+   ## Configure and link your App Clips
+
+   https://developer.apple.com/videos/play/wwdc2020/10146/
+
+   _Ada Chan, Luming Yin_
+
+   Duration: 22:56
```
Comment (rogerluan) [inline README.md]: Should we add this to every video? Thoughts?

Comment (Blackjacx) [reply README.md]: I updated the contribution rules: https://github.com/Blackjacx/WWDC/blob/master/CONTRIBUTING.md#submitting-changes

File: README.md [hunk 2]
```diff
@@ -421,4 +422,52 @@ _Matt Ricketson, Taylor Kelly_
...
\ No newline at end of file
+   - Other Frameworks: **AuthenticationServices**, **AVKit**, **MapKit**, **SceneKit**, **SpriteKit**
+
+## Configure and link your App Clips
+
+https://developer.apple.com/videos/play/wwdc2020/10146/
+
+_Ada Chan, Luming Yin_
+
+Duration: 22:56
```
Comment (Blackjacx) [inline README.md]: Please without the duration, see the new https://github.com/Blackjacx/WWDC/blob/master/CONTRIBUTING.md

File: README.md [hunk 3]
```diff
@@ -421,4 +422,52 @@ _Matt Ricketson, Taylor Kelly_
    - `onDrag {}` and `onDrop {}` to support drag & drop
    - `UniformTypeIdentifers` framework for e.g. introspection of file types
    - `Sign in with Apple` as first class component in SwiftUI • available on every platform
-   - Other Frameworks: **AuthenticationServices**, **AVKit**, **MapKit**, **SceneKit**, **SpriteKit**
\ No newline at end of file
+   - Other Frameworks: **AuthenticationServices**, **AVKit**, **MapKit**, **SceneKit**, **SpriteKit**
+
+## Configure and link your App Clips
+
+https://developer.apple.com/videos/play/wwdc2020/10146/
```
Comment (Blackjacx) [inline README.md]: If possible please use the short url version, it is easier to parse later on for me :-) You can just copy & paste one of the others and change the session number.

**Review by Blackjacx — CHANGES_REQUESTED** [review]
Thanks for your input, really cool addition 🙏

File: README.md (hunk 1 above)
Comment (rogerluan) [reply README.md]: Sweet, that works for me! 💪

**Comment by rogerluan** [comment]
Sorry for putting all the fixes in a single merge commit 🤦 but I addressed your comments, fixed some typos in the new latest and removed redundant trailing whitespaces at the end of the file. :)

**Review by Blackjacx — APPROVED** [review]
All good bro, you are my lifesaver! Thanks for all the corrections. This is really nice and keeps up the quality of the notes 👍

**Pull request merged**